src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double longerSIde;
```

Comment:
The variable name 'longerSIde' should be in camel case format. Consider renaming it to 'longerSide' to follow Java naming conventions.